### PR TITLE
Explain weights and predictions with eli5

### PR DESCRIPTION
**Pull request by lopuhin**
Add two scripts, ``explain-weights`` and ``explain-prediction``. They both require a sample of crawled documents in CDR format in order to fit the hashing vectorizers. Only ``link_vectorizer`` is supported, ``page_vectorizer`` is not supported yet.

The link explanations look like this:
screenshot from 2017-05-03 14-04-57
(hashingvectorizer-3 is the page url here)

### CODE REVIEW

**Comment by lopuhin** [comment]
Maybe it would better to merge this as one commit, cause the branch contains a lot of back and forth and code that was moved into eli5 feature union support.

**Comment by kmike** [comment]
Does it need a new eli5 release, so that we can update deep-deep requirements?

**Comment by lopuhin** [comment]
Oh right, I forgot that the changes are not yet released, and that eli5 is not in requirements, thanks for the catch!
I'll add eli5 to requirements, and then we can merge this PR after the next eli5 release.

**Comment by kmike** [comment]
A very hepful feature, thanks @lopuhin!
